### ISSUE TITLE
Simplify main README

### ISSUE THREAD

**Issue by Nana-EC** [hashgraph/hedera-local-node#884]
### Problem

The currnet README is long and detailed, with a mixture of nom and docker commands and focus.
This can be confusing and unclear

### Solution

Let's simplify the README with a clear focus on the npm approach first and the docker approach as pone for power users

- [ ] Move docker requirements setup into a separate README with a link in the main README
- [ ] Streamline install and run steps to be npm only
- [ ] Move docker install and run step into separate README and link in main README
- [ ] Add a section for common issues that link to a separate README with common issues that users may hit on setup - usually due to not following instructions or because of left over artifacts from other docker sessions

The goal is for a simplified experience with extra details readily available

**Comment by stastix** [comment]
I agree that while the readme provides wealth of information I believe it has become e a bit cluttered and challenging to read. I'll gladly take this issue I believe it should be divided into something like this  
```plaintext
  ├── Docker.md
  ├── CLI.md
  ├── Environment-Variables.md
  ├── Grafana-Prometheus.md
  ├── FAQ.md
README.md```

**Comment by nadezhdapopovaa** [comment]
@georgi-l95 could you assign me?
